source/disassemble.cpp

```diff
@@ -49,6 +49,8 @@ namespace {
 class Disassembler {
   enum { kStandardIndent = 15 };
 
+  using out_stream = libspirv::out_stream;
```

Comment:
Yeah, both my suggestions are impossible. :)  Sorry!

For the record: http://en.cppreference.com/w/cpp/language/using_declaration#In_class_definition